### ISSUE TITLE
Genotype Info from Target

### ISSUE THREAD

**Issue by and02709** [choishingwan/PRSice#127]
Hi, 
I have a question about the "Loading Genotype info from target"  section that appears in the log file.

I am using a GWAS base that has 9733 variants observed in the base file, but several are ambiguous and are excluded.  This leaves 8234 total variants included from the base file.

In the following section where it lists the information under the header "Loading Genotype info from target", it lists that I have 588 people (289 male, 299 female) observed
346 founders included

568 variants included.  (*** this number is my concern, this is much smaller than the variant number in the base file)

When I examined similar output for the TOY data set provided, it reads,
91062 variants observed in base file
88836 total variants included from base file

Under Loading Genotype info from target it lists
2000 people (1024 male, 976 female) observed
2000 founders included

88836 variants included.  (*** this is exactly the number in the base file)

Ultimately, I am not sure why so few of my genetic variants from base file appear to be included.  In your experience, does this suggest a problem that I should consider, or is easily explained somehow?  I think I am missing something important - I guess I'm asking for advice.

Thank you.

Sincerely,
Michael

**Comment by choishingwan** [comment]
You need to check if the same SNPs are found in both the target and base.
You can check that by examining their SNP ID. There are two issues here:
You base is relatively small, usually we will have many more SNPs in the
base. Did you only include significant SNPs? Second, based on the low
number of SNPs found in target, it seems like the SNP ID in your target
doesn't match those in the base. You can consider using a denser base or
impute your target


Dr Shing Wan Choi
Postdoctoral Fellow
Genetics and Genomic Sciences
Icahn School of Medicine, Mount Sinai, NYC

**Comment by misrak** [comment]
Hello!
I had the same question actually about the difference. I am using a base and target file with both same common SNPs and both the files have identical SNP ID. Is there any other reason why the difference would still be there?
-------------------------------
Example: 

Start processing common631SNP_ADD.assoc
==================================================


Reading 100.00%
Base file:
../association_results/common631SNP_ADD.assoc.logistic
631 variant(s) observed in base file, with:
13 variant(s) located on haploid chromosome
26 NA stat/p-value observed
618 total variant(s) included from base file



Loading Genotype info from target
==================================================

181 people (119 male(s), 62 female(s)) observed
181 founder(s) included

533 variant(s) included


Thank you so much.
Kaalindi

**Comment by choishingwan** [comment]
Can you try running with version 2.2.5 and send me the log? That should give me more information.

**Comment by misrak** [comment]
[new.log](https://github.com/choishingwan/PRSice/files/3456905/new.log)

I hope this helps.

**Comment by choishingwan** [comment]
Your target file doesn't contain 631 variants. There are only 533 variants in your target.

**Comment by misrak** [comment]
No, I have checked it multiple times it contains 631 snps. 
you can it yourself. would you also need covariate file or this will do?
[base_target_input_files.zip](https://github.com/choishingwan/PRSice/files/3458516/base_target_input_files.zip)

**Comment by choishingwan** [comment]
Now I see. As your base only contain one allele (A1), ambiguous SNP removal
cannot be performed in the base data. When we go into target data loading,
we observed there are ambiguous SNPs, which we then removed (a total of
76). Due to some problem on my side, this message wasn't reported, thus
lead to the confusion. This problem should be solved in future release

Dr Shing Wan Choi
Postdoctoral Fellow
Genetics and Genomic Sciences
Icahn School of Medicine, Mount Sinai, NYC
